tests/integration/docker/exec.bats

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bats
```

Comment:
You use both ubuntu and busybox images, could we just have one (say busybox?) no need to download both.